packages/oui-pagination/_mixins.less

```diff
@@ -0,0 +1,139 @@
+@import '../oui-button/_variables';
+@import './_variables';
+
+#oui {
+  .pagination-base(
+    @button-selector,
+    @button-link-selector,
+    @button-secondary-selector,
+    @button-group-selector
+  ) {
+    #oui > .base-font();
+
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: @oui-pagination-padding;
+    font-size: @oui-pagination-font-size;
+
+    /* stylelint-disable indentation */
```

Comment:
Why do you have to disable stylelint ? :(

Reply:
Because `stylelint` is not able to check the indentation correctly inside mixin object. If we don't disable it, the expected indentation is completly offset and broken. `stylelint` is mainly for `sass`, so some wrong behaviors happens on `less` validator.